test/functional/ui/completion_spec.lua

```diff
@@ -0,0 +1,41 @@
+local helpers = require('test.functional.helpers')
+local Screen = require('test.functional.ui.screen')
+local clear, feed, execute = helpers.clear, helpers.feed, helpers.execute
+local eval, eq, source = helpers.eval, helpers.eq, helpers.source
+
+describe("completion", function()
+  local screen
+
+  before_each(function()
+    clear()
+    screen = Screen.new(25, 5)
+    screen:attach()
+  end)
+
+  after_each(function()
+    screen:detach()
+  end)
+
+  describe("v:completed_item", function()
```

Comment:
I cannot find error/exception helpers in helpers.lua.
I think it is not the PR work.